code/messaging/src/test/resources/robolectric.properties

```diff
@@ -0,0 +1 @@
+sdk=34
```

Comment:
Can we configure this specifically for `MessagingRichPushBuilderTests` instead of making it a global setting? Its causing some of the content card snapshot tests to fail since it was running on SDK 33 earlier

Reply:
sure, i'll remove this file and can set API versions on a per test basis

Reply:
Hmm looks like that didn't fix the failing tests. It might be because of the updated roboelectric version. We can merge this PR and update the baseline screenshots after we merge the branch to main

Reply:
sounds good, i'll kick off the workflow once its merged